src/bindgen/index.md

```diff
@@ -48,16 +48,17 @@ Then add `bindgen` as a dependency and tell `Cargo` about our build script
 
 `Bindgen` works by reading a `C`-style header file using `libclang`, then
 it runs the preprocessor and creates bindings based on the output. Therefore, 
-before we can start our build script we'll need a dummy header which pulls in
-the `bzip2` library installed on your system.
+the build script needs a dummy header which pulls in the `bzip2` library
+installed on your system.
```

Comment:
Seems I need to learn how to add downloaded libraries and header files to the PATH variable.
```
--- stdout
cargo:rustc-link-lib=bz2
wrapper.h:1:10: fatal error: 'bzlib.h' file not found, err: true

--- stderr
wrapper.h:1:10: fatal error: 'bzlib.h' file not found
thread 'main' panicked at 'Unable to generate bindings: ()', src\libcore\result.rs:860
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```